### PR TITLE
Witivio Connector V2

### PR DESCRIPTION
**Pull request by tdechanterac**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Witivio/apiDefinition.swagger.json [hunk 1]
```diff
@@ -3,7 +3,12 @@
   "info": {
     "title": "Witivio",
     "description": "Witivio 365 and Microsoft Flow.",
```
Comment (sriyen-msft) [inline certified-connectors/Witivio/apiDefinition.swagger.json:5]: @tdechanterac As per the submission you made in the ISV Studio, it should be - `"description": "Build advanced virtual agent with the power of Witivio 365"`

File: certified-connectors/Witivio/apiDefinition.swagger.json [hunk 2]
```diff
@@ -3,7 +3,12 @@
...
-    "version": "1.0"
+    "version": "2.0",
+    "contact": {
+      "name": "Witivio",
+      "url": "https://witivio.com",
```
Comment (sriyen-msft) [inline certified-connectors/Witivio/apiDefinition.swagger.json:9]: @tdechanterac Please add domain to your website, like below:

`"url": "https://www.witivio.com"`

File: certified-connectors/Witivio/apiDefinition.swagger.json [hunk 3]
```diff
@@ -1194,5 +1262,19 @@
       "oauth2_auth": []
     }
   ],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://witivio.com"
```
Comment (sriyen-msft) [inline certified-connectors/Witivio/apiDefinition.swagger.json:1268]: @tdechanterac Please add domain to your website, like below:
`"https://www.witivio.com"`

File: certified-connectors/Witivio/apiDefinition.swagger.json [hunk 4]
```diff
@@ -1194,5 +1262,19 @@
...
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://witivio.com"
```
Comment (sriyen-msft) [inline certified-connectors/Witivio/apiDefinition.swagger.json:1272]: @tdechanterac Please add domain to your website, like below:
`"https://www.witivio.com"`

File: certified-connectors/Witivio/apiProperties.json [hunk 5]
```diff
@@ -1,25 +1,47 @@
...
-            "hidden": "true",
-            "required": "false"
+            "required": "false",
+            "hidden": "true"
           }
         }
       }
     },
-    "iconBrandColor": "#8F3163"
+    "iconBrandColor": "#007ee5",
```
Comment (sriyen-msft) [inline certified-connectors/Witivio/apiProperties.json:42]: @tdechanterac The iconBrandColor should be `"#8F3163"` as per the submission in ISV studio.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Witivio/apiDefinition.swagger.json (hunk 1 above)
Comment (tdechanterac) [reply certified-connectors/Witivio/apiDefinition.swagger.json]: Thank you. I've made the proper change.

File: certified-connectors/Witivio/apiDefinition.swagger.json (hunk 2 above)
Comment (tdechanterac) [reply certified-connectors/Witivio/apiDefinition.swagger.json]: I've modified as well all the links with the proper links.

**Comment by tdechanterac** [comment]
@sriyen-msft we have updated all your comment, What is the next step ?

File: certified-connectors/Witivio/apiProperties.json [hunk 6]
```diff
@@ -1,25 +1,24 @@
 {
```
Comment (sriyen-msft) [inline certified-connectors/Witivio/apiProperties.json:1]: @tdechanterac The apiProperties.json file is outdated one. Please provide the latest one you have uploaded in the ISV Studio.

`{
    "properties": {
        "capabilities": [
            "actions",
            "triggers"
        ],
        "connectionParameters": {
            "token": {
                "oAuthSettings": {
                    "clientId": "9275fab6-86e0-4fb6-affa-1473100e8f22",
                    "customParameters": {
                        "loginUri": {
                            "value": "https://login.windows.net"
                        },
                        "resourceUri": {
                            "value": "https://witivio.com/internalapi"
                        },
                        "tenantId": {
                            "value": "common"
                        }
                    },
                    "identityProvider": "aad",
                    "properties": {
                        "AzureActiveDirectoryResourceId": "https://witivio.com/internalapi",
                        "IsFirstParty": "False"
                    },
                    "redirectMode": "Global",
                    "redirectUrl": "https://global.consent.azure-apim.net/redirect",
                    "scopes": [
                    ]
                },
                "type": "oauthSetting"
            },
            "token:TenantId": {
                "metadata": {
                    "sourceType": "AzureActiveDirectoryTenant"
                },
                "type": "string",
                "uiDefinition": {
                    "constraints": {
                        "hidden": "true",
                        "required": "false"
                    }
                }
            }
        },
        "iconBrandColor": "#8F3163",
        "publisher": "Witivio",
        "stackOwner": "Witivio"
    }
}`

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@tdechanterac   I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by sriyen-msft** [comment]
@tdechanterac I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by sriyen-msft** [comment]
@tdechanterac I am abandoning the PR as there is no response from your side since last two weeks. You can reopen the PR when you are ready for certification.

**Pull request closed without merging**